CODEOWNERS

```diff
@@ -282,6 +282,7 @@ Makefile* @cilium/build
 /bpf/lib/ids.h @cilium/loader
 /bpf/lib/proxy.h @cilium/proxy @cilium/sig-datapath
 /bpf/include/bpf/tailcall.h @cilium/loader
+/bpf/include/bpf/config @cilium/loader
```

Comment:
Please also add `CODEOWNERS` entries for `tools/dpgen` and `contrib/scripts/check-datapathconfig.sh`

Reply:
Thanks, fixed!